### ISSUE TITLE
[high] JAWS+Chrome: Preferences: Toggle button announced without label, role or state

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5529]
## Action Performed:
Prerequisite: the user is logged in.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Navigate to Settings > Preferences.
3. Once the Preferences overlay opens, press Tab key to focus the toggle button 'Receive relevant feature updates and Expensify news' and listen to the announcement.

## Expected Result:
The element should announce its label, role and state: 'Receive relevant feature updates and Expensify news, toggle button, checked'.

## Actual Result:
The element does not announce its label, role or state.

HTML:
`<div class="css-1dbjc4n" style="flex-direction: row; justify-content: space-between; margin-bottom: 24px; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -webkit-box-pack: justify;"><div class="css-1dbjc4n" style="flex: 4 1 0%;"><div dir="auto" class="css-901oao" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px;">Receive relevant feature updates and Expensify news</div></div><div class="css-1dbjc4n r-13awgt0" style="align-items: flex-end; -webkit-box-align: end;"><div tabindex="0" class="css-1dbjc4n r-1loqt21 r-1777fci r-1otgn73 r-1i6wzkk r-lrvibr" style="background-color: rgb(3, 212, 124); border-radius: 20px; height: 28px; padding: 15px; transition-duration: 0s; width: 50px;"><div class="css-1dbjc4n r-u8s1d" style="background-color: rgb(255, 255, 255); border-radius: 11px; height: 22px; left: 4px; transform: translateX(20px); width: 22px;"></div></div></div></div>`

## Workaround:
No workaround

## Area issue was found in:
Preferences overlay

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader users are not informed about the purpose and functionality of the focused element.

## Suggested resolution:
Implement an accessible toggle button to announce its role and state. Use the `<label>` tag with "for" attribute to associate the visible label 'Receive relevant feature updates and Expensify news' with the button. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Switch_role

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134781071-3272a498-a20e-4ad3-9db7-dbcdb0517f73.mp4

![Bug5183741_Name__role__value_-_toggle_button_-_preferences](https://user-images.githubusercontent.com/88733897/134781073-fda81236-737d-4ec8-b6ea-a8966fc0f6dc.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
